Makefile

```diff
@@ -77,6 +77,12 @@ install-key:
 	@go install ./cmd/cel-key
 .PHONY: install-key
 
+## docgen: Build docgen binary.
+docgen:
+	@echo "--> Building docgen"
+	@go build ./cmd/docgen
+.PHONY: docgen
```

Comment:
Is there a use case for this binary independently?

Reply:
No, actually not. I was also on the fence about including this. Should I remove it?

Reply:
Let's remove it, yes

Reply:
done